docs/source/modules.rst

```diff
@@ -252,6 +254,36 @@ The read lengths input to `groot`_ must conform to the settings used during
 `groot`_ database construction. The length window can be configured in the
 config file.
 
+AMRPlusPlus_v2
+-------
+:Tool: `amrplusplus`_
+:Output folder: ``amrplusplus``
+
+`amrplusplus`_ will align reads to `megares`_ antibiotic resistance gene database to
+produce antibiotic resistance gene profiles. Output will structured as::
+
+        ├ AlignToAMR
+        │   └ <sample>.amr.alignment.sam
+        ├ RunResistome
+        │   ├ <sample>.class.tsv
+        │   ├ <sample>.gene.tsv
+        │   ├ <sample>.group.tsv
+        │   └ <sample>.mech.tsv
+        ├ ResistomeResults
+        │   └ AMR_analytic_matrix.csv
+        ├ RunRarefaction
+        │   ├ <sample>.class.tsv
+        │   ├ <sample>.gene.tsv
+        │   ├ <sample>.group.tsv
+        │   └ <sample>.mech.tsv
+
+``AMR_analytic_matrix.csv`` contains aggregated results of gene counts for all samples 
+aligned against `megares`_, based on the threshold set in ``config.yaml``. Pasting a gene name 
+or accession number into the database will provide detailed information and links to 
+published papers.
+
+`amrplusplus`_ can only be run with the ``--use-singularity`` setting, meaning it can not be 
```

Comment:
```suggestion
`amrplusplus`_ can only be run with the ``--use-singularity`` setting, meaning it cannot be 
```

Reply:
Haven't tried it but It would probably complain that that a specific tool is unrecognizable or doesn't understand some command.